Fix vector tiles

gchoqueux opened the pull request:
## Description

in TMS/VectorTile, wrong projection convert for globe
Because, vector tile's projection is 'EPSG:4326' and texture's projection is 'EPSG:3857'

## Screenshots
**after fix**
![vector tile](https://user-images.githubusercontent.com/11291849/43468820-28b98458-94e5-11e8-94c8-d2d23a9444fa.png)
**before fix**
![image](https://user-images.githubusercontent.com/741255/42682706-a07d9000-868b-11e8-887c-d5c89fe2f5ad.png)

zarov commented:
Thanks for this ! The failed CI comes from the change of the return value, which impacts other parts of itowns. That change is necessary, as there seems to be a bug with `babel` (but I can't find what is it really), so I'll propose another structure of the returning value in `GeojsonParser` tomorrow.

gchoqueux committed:
fix(Protocol): in TMS/VectorTile, wrong projection convert for globe

Because, vector tile's projection is 'EPSG:4326' and texture's projection is 'EPSG:3857'

zarov reviewed (approved):
Thanks for the fix, it looks really better this time !

gchoqueux committed:
fix(parser): problem filtering in geoJsonParser

gchoqueux committed:
fix(example): define pole texture for globe vector tile and globe max level subdivision
